en/messages.json

```diff
@@ -211,8 +211,8 @@
   "enableBookMarkMenus": {
     "message": "Enable Bookmark Menus"
   },
-  "firefoxAccountsSync": {
-    "message": "Firefox accounts sync:"
+  "sync": {
+    "message": "Sync:"
```

Comment:
Is there a specific reason to not use `Synchronization`? 

We can use Sync here, but that's just short for Synchronization when the space is limited, is not a feature anymore
https://acorn.firefox.com/latest/content/word-list/s-t-D8DB7Wjd

Reply:
I wasn't sure if we also changed the way to refer to Sync (as in Firefox Sync). So do you think using "Synchronization" for the title and "Enable synchronatization" for the permission string would be better in this case?

Reply:
I think it would make sense (look out for typos, `Enable synchronization`).